packages/features/bookings/components/event-meta/Details.tsx

```diff
@@ -112,16 +111,6 @@ export const EventDetails = ({ event, blocks = defaultEventDetailsBlocks }: Even
         }
 
         switch (block) {
-          case EventDetailBlocks.DESCRIPTION:
```

Comment:
Removed from this default event meta block, because it's now shown on top of all other event data, which is outside the wrapping div this component is in.